### PR TITLE
Adding Course  Lesson Player

### PR DESCRIPTION
**Pull request by adarsh-technocrat**
**Describe the changes you've made**

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

**Provide a Deployed link of route/page that needs to review**
Preview: Deploy preview link here with the appropriate route

<!-- Please add deployed link to here text >

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
Is this PR complete?

**Comment by adarsh-technocrat** [comment]
> Is this PR complete?

No @Abhishek-kumar09  there is some responsive issue  I'm working on it so it will finish after that !.

**Comment by adarsh-technocrat** [comment]
Hey @Abhishek-kumar09  PR is up for Review =)

File: src/components/LessonsPlayerComponents/SubComponents/VideoContentList.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        {[1, 2, 3].map((items, index) => (
+          <VideoSubSectionList key={index} />
+        ))}
+      </Collapse>
+    </Box>
+  );
+}
+
+const useStyles = makeStyles((theme) => ({
+  ListContainer: {
```
Comment (Abhijay007) [inline src/components/LessonsPlayerComponents/SubComponents/VideoContentList.js:59]: camel Casing

File: src/components/LessonsPlayerComponents/SubComponents/VideoSubSectionList.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    </ListItem>
+  );
+}
+
+const useStyles = makeStyles((theme) => ({
+  ListContainer: {
+    flex: 2,
+    height: 800,
+    marginLeft: theme.spacing(3),
+  },
```
Comment (Abhijay007) [inline src/components/LessonsPlayerComponents/SubComponents/VideoSubSectionList.js:43]: defined but maybe not used

**Pull request merged**